walkthroughs/howto-k8s-ipv6/README.md

```diff
@@ -0,0 +1,114 @@
+## Overview
+In this walk through we'll be setting up applications with different IP version capabilities and configuring App Mesh resources.
+
+## IP Preferences in Mesh
+With the introduction of IPv6 support in App Mesh a new IP preference field has been added to mesh. IP preferences will impact how Envoy configuration gets generated. The possible values for IP preferences in Mesh are the following.
+
+Since EKS is single stack, an EKS cluster can be either IPv6 or IPv4. The IpPreference of a Mesh can have two possible values:
+
+- IPv4_ONLY: only use IPv4
+- IPv6_ONLY: only use IPv6
+
+This field is not a required setting for mesh. Users could have No Preference by not specifying the field. 
+
+For an IPv6 cluster, the mesh ip preference is by default to IPv6. 
+
+Meshes: Adding an IP preference to a mesh impacts how Envoy configuration will be generated for all virtual nodes and virtual gateways within the mesh. A sample mesh spec that includes an IP preference can be seen below.
+
+```
+"spec": {
+   "serviceDiscovery": {
+   "ipPreference": "IPv6_ONLY"
+   }
+}
+```
+
+## Prerequisites
+1. To use the IPv6 feature create EKS cluster with ipFamily as IPv6. Sample IPv6 yaml file: [IPv6 EKS cluster](ipv6-cluster.yaml)
+2. Install appmesh-controller.
+
+Note: Run the following to check that cluster name is set.
+
+       kubectl get deployment -n appmesh-system appmesh-controller -o json | jq -r '.spec.template.spec.containers[].args[] | select(contains("cluster-name"))'
+
+   You should get as output:
+
+   *--cluster-name=<name-of-your-cluster>*
```

Comment:
In markdown `*--cluster-name=<name-of-your-cluster>*` shows up like below
*--cluster-name=<name-of-your-cluster>*

Maybe you want to fix this markdown here?